TVC_MVP/County.swift

```diff
@@ -18,17 +18,13 @@ struct County {
         self.countyID = countyID
         self.countyName = countyName
     }
-
-    func hasExtras() -> Bool {
-        return countyExtras != nil
-    }
     
 }
 
 struct CountyExtras {
 
-    var population: Int?
```

Comment:
This should stay optional, as the values might be not there. We don't want to have `0` as default.